development/roles/foreman_development/tasks/main.yaml

```diff
@@ -0,0 +1,232 @@
+---
+- name: Enable nodejs module stream
+  ansible.builtin.command:
+    cmd: dnf module enable -y nodejs:{{ foreman_development_nodejs_stream | default('22') }}
+  register: foreman_development_nodejs_module_result
+  changed_when: "'CHANGED' in foreman_development_nodejs_module_result.stdout"
+  failed_when:
+    - foreman_development_nodejs_module_result.rc != 0
+    - "'Nothing to do' not in foreman_development_nodejs_module_result.stdout"
+    - "'already enabled' not in foreman_development_nodejs_module_result.stdout"
+
+- name: Install nodejs from enabled module
+  ansible.builtin.package:
+    name: nodejs
+    state: present
+
+- name: Enable CodeReady Builder repository
+  community.general.rhsm_repository:
+    name: "codeready-builder-for-rhel-{{ ansible_distribution_major_version }}-{{ ansible_architecture }}-rpms"
+    state: enabled
+  when: ansible_distribution == 'RedHat'
+
+- name: Install development packages
+  ansible.builtin.package:
+    name: "{{ foreman_development_packages }}"
+    state: present
+
+- name: Disable firewall for development environment
+  ansible.builtin.systemd:
+    name: firewalld
+    state: stopped
+    enabled: false
+  failed_when: false
+
+- name: Create development user
```

Comment:
Since we would like this to work as root by default on a machine that has no users, I would suggest when the user is `root`:

```
when: foreman_development_user != "root"
```

To avoid: 

>      TASK [foreman_development : Create development user]
>     ***************************
>   - 'fatal: [localhost]: FAILED! => {'
>   - '    "changed": false,'
>   - '    "name": "root",'
>   - '    "rc": 8'
>   - '}'
>   - ''
>   - 'MSG:'
>   - ''
>   - 'usermod: user root is currently used by process 1'
>   - ''
>   - ''
>   - >-